stable/testlink/templates/deployment.yaml

```diff
@@ -18,9 +18,13 @@ spec:
     spec:
       {{- if .Values.image.pullSecrets }}
       imagePullSecrets:
-      {{- range .Values.image.pullSecrets }}
+        {{- range .Values.image.pullSecrets }}
```

Comment:
Isn't affinity the preferred way to do node-selection nowadays? https://kubernetes.io/docs/concepts/configuration/assign-pod-node/

Reply:
Yes, I was implementing it as per the issue. I'll be revisit this a bit later to consider supporting both since I'm already doing the work for this.

Thanks!